Riot/Modules/MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -1872,10 +1872,11 @@ - (void)sendTextMessage:(NSString *)text success:(void (^)(NSString *))success f
 
 - (void)sendReplyToEventWithId:(NSString*)eventIdToReply
                withTextMessage:(NSString *)text
+          actualRoomDataSource:(MXKRoomDataSource *)actualData
                        success:(void (^)(NSString *))success
                        failure:(void (^)(NSError *))failure
```

Comment:
It is slightly confusing to pass in one data source to another here (I can totally see why though).

As these two updated methods both only have a single call from the `RoomViewController`, if it works, it might be nicer to move the `eventWithEventId:` call up one level and then pass in the event to this method rather than another data source. What do you think?
```suggestion
- (void)sendReplyToEvent:(MXEvent *)eventToReply
         withTextMessage:(NSString *)text
                 success:(void (^)(NSString *))success
                 failure:(void (^)(NSError *))failure
```

Reply:
Yeah, it make more sens to do it this way, it's done